### DISCUSSION TITLE
How to change Globally Shared Properties

### DISCUSSION THREAD

**Discussion by ikondrasovas** [dotnet/docfx#7948] [Q&A]
On the template system introduction tutorial, down at the end there is a topic about Globally Shared Properties.

I would like to modify for example __global.backToTop and localize it to a specific language, but I could not make it at the end.

Can you please extend a bit the explanation (maybe with an example) on how to have and use this global.json file?

Thanks!

**Comment by cedar-ave** [comment]
If this meets what you're asking...

Run `docfx template export` and go to _exported_templates/default/token.json.
Put a copy of token.json at the root of your template (which, when added, probably will look something like templates/your-template/token.json). This overrides the token.json in the `default` template, as long as your docfx.json looks something like:
```
"template": [
    "default",
    "templates/your-template"
]
```
In token.json is a line `backToTop: Back to top`. Change `Back to top` to whatever your want.
